src/components/app/style.css

```
@@ -73,7 +73,7 @@ smoothly-app>smoothly-notifier>header>nav>ul>li>*:not(a) {
 	margin: 0;
 }
 smoothly-app>smoothly-notifier>header>h1 {
-	margin-left: 60px;
+	margin-left: 3.333rem;
```

Don't need to be that precise
```suggestion
	margin-left: 3.5rem;
```